src/core/components/pin-set.js

```diff
@@ -6,8 +6,8 @@ const protobuf = require('protons')
 const fnv1a = require('fnv1a')
 const varint = require('varint')
 const { DAGNode, DAGLink } = require('ipld-dag-pb')
-const some = require('async/some')
-const eachOf = require('async/eachOf')
+const someSeries = require('async/someSeries')
+const eachOfSeries = require('async/eachOfSeries')
```

Comment:
What does this change do?

Reply:
Using `some` is making a lot of parallel dht find and connect requests. 

So, everything will get queried even if we find the target node, as the find takes longer than initiating the requests. 

In the long run, we should refactor the `pin-set`, in order to try to parallelize some operations, but this is the safest option currently.